aardwolf-templates/templates/home/feed.rs.html

```diff
@@ -16,7 +16,7 @@
             <strong>Barbara Middleton</strong>
             <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis porta eros lacus, nec ultricies elit blandit non. Suspendisse pellentesque mauris sit amet dolor blandit rutrum. Nunc in tempus turpis.
             <br>
-            <small><a>@i18n!(catalog, "Like")</a> · <a>@i18n!(catalog, "Reply")</a> · <a>@i18n!(catalog, "Boost")</a> · <i class="fa fa-ellipsis-h" aria-hidden="true"></i> · 3 hrs</small>
+            <small><a>@i18n!(catalog, "Like")</a> · <a>@i18n!(catalog, "Reply")</a> · <a>@i18n!(catalog, "Boost")</a> · <span class="fa fa-ellipsis-h" aria-hidden="true"></span> · 3 hrs</small>
```

Comment:
why are we changing Icons from &lt;i&gt; to &lt;span&gt;?

also you should be using `@:icon("ellipsis-h")` anyway

Reply:
best practices is to use span instead of the "italic" tag.